app/components/back-button.tsx

```diff
@@ -36,13 +31,15 @@ export const BackButton: FC<Omit<ButtonProps, 'children'>> = ({ onClick, ...prop
         handleBack();
         onClick?.(e);
       }}
+      pointerEvents={!hasBackState ? 'none' : 'all'}
       isDisabled={!hasBackState}
```

Comment:
![image](https://user-images.githubusercontent.com/1618764/115521080-13b95900-a28b-11eb-8837-2f4d3a5cb38b.png)

this causes this runtime error

Reply:
ah yeah should be removed, or we can do `as={Button}`